lib/whitehall/asset_manager_and_quarantined_file_storage.rb

```diff
@@ -25,6 +25,10 @@ def url
       @quarantined_file.url
     end
 
+    def asset_manager_path
+      @asset_manager_file.path
```

Comment:
I wonder if we should add an accessor for `legacy_url_path` to AssetManagerStorage and delegate to that instead? If I understand the commit message correctly, you will be eventually be looking up an asset in asset manager using the `legacy_url_path` passing in this method?

Reply:
I'm slightly unsure what you mean here. Do you mean something like this:

`AttachmentUploader#legacy_url_path` -> `Whitehall::AssetManagerAndQuarantinedFileStorage::File#legacy_url_path` -> `Whitehall::AssetManagerFileStorage::File#legacy_url_path`

If so, I'm not sure this is clearly better and in any case, since I expect these `asset_manager_path` methods to go away, so I think I'm going to merge this as it stands so I can make progress. Please feel free to add more explanatory comments here or elsewhere if you think it would be worth making a change.